src/components/app-cta/app-cta.jsx

```diff
@@ -36,9 +43,15 @@ var AppCta = React.createClass({
     this.setState({
       loading: true
     });
+
+    if (!this.isAndroid()) {
+      return window.location = AppCta.FALLBACK_URL;
+    }
+
     this.timeout = setTimeout(function () {
       window.location = AppCta.FALLBACK_URL;
```

Comment:
Looks like the play store URL still opens in the background once you re-focus back to your browser on an Android device. Seems like that shouldn't be the case...